app/views/application/_datetime_field.html.erb

```diff
@@ -1,5 +1,6 @@
+<%- disabled = local_assigns.fetch(:disabled?, false)%>
 <div>
   <%= form.label attribute %>
-  <%= form.datetime_field attribute %>
+  <%= form.datetime_field attribute, disabled: disabled %>
```

Comment:
I feel like this could be merged independently, and maybe wombo-comboed with a `FieldComponent` so more of our `:x_field` partials have standardized inputs.

Reply:
Wombo-comboed! That's a new one for me.